sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -356,6 +356,7 @@ public abstract partial class ClientOptions
         public static Azure.Core.ClientOptions Default { get { throw null; } }
         public Azure.Core.DiagnosticsOptions Diagnostics { get { throw null; } }
         public Azure.Core.RetryOptions Retry { get { throw null; } }
+        public Azure.Core.Pipeline.HttpPipelinePolicy? RetryPolicy { get { throw null; } set { } }
```

Comment:
Curious: Why is this on ClientOptions rather than RetryOptions?

Have we given consideration to how we might replace other "fixed" policies in the pipeline?  For example, the redirect or buffering policies?  Would we add them at this same level on ClientOptions?

Reply:
> Curious: Why is this on ClientOptions rather than RetryOptions?

RetryOptions is currently protocol-agnostic and I'd like to keep it that way if possible.


> Have we given consideration to how we might replace other "fixed" policies in the pipeline? For example, the redirect or buffering policies? Would we add them at this same level on ClientOptions?

Good question - I think we would use the same approach for replacing other policies that we choose to expose.

Reply:
We explored this question here - https://github.com/Azure/azure-sdk-for-net/pull/31892#discussion_r999940604

The consensus was that using a single property is preferable as we don't have a lot of reason to expect the set of replaceable policies to grow beyond this one.